### PR TITLE
Update txt2tags.py

### PR DESCRIPTION
**Pull request by goetzschnell**
changes to avoid warnings from w3 validator, tested with file sample.html, tested for options --enum-title and --toc too
topics to discuss eventually:
- insertion of language code,
- header levels,
- other solutions for id=body etc.

### CODE REVIEW

File: txt2tags.py [hunk 1]
```diff
@@ -82,7 +82,7 @@
 my_url = "https://txt2tags.org"
 my_name = "txt2tags"
 my_email = "jendrikseipp@gmail.com"
-__version__ = "3.7"
+__version__ = "3.7.2"
```
Comment (jendrikseipp) [inline txt2tags.py]: No need to change the version. I'll do that before the next release.

File: txt2tags.py [hunk 2]
```diff
@@ -235,13 +235,13 @@
...
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
 <meta charset="%(ENCODING)s">
 <title>%(HEADER1)s</title>
 <meta name="generator" content="https://txt2tags.org">
 <link rel="stylesheet" href="%(STYLE)s">
-<style type="text/css">
+<style>
```
Comment (jendrikseipp) [inline txt2tags.py]: Is it discouraged to add the CSS type explicitly?

File: txt2tags.py [hunk 3]
```diff
@@ -272,7 +272,7 @@
 <h3>%(HEADER3)s</h3>
 </hgroup>
 </header>
-<article>
+
```
Comment (jendrikseipp) [inline txt2tags.py]: Why not use "article"?

File: txt2tags.py [hunk 4]
```diff
@@ -561,13 +561,13 @@ def getTags(config):
             "tableTitleCellClose": "</th>",
             "tableTitleCellOpen": "<th~s~>",
             "title1Close": "</section>",
-            "title1Open": "<section~A~>\n<h1>\a</h1>",
+            "title1Open": "<section~A~>\n<h2>\a</h2>", #www3 validator: h1 only for top level
```
Comment (jendrikseipp) [inline txt2tags.py]: I believe this is a warning and not an error in the w3 validator. I think it's nice to map the titles to h1-h5, so let's ignore the warning and please undo this change.

File: txt2tags.py (hunk 2 above)
Comment (goetzschnell) [reply txt2tags.py]: There is a warning:
-----------------------------------------
Warning: The type attribute for the style element is not needed and should be omitted.

From line 7, column 1; to line 7, column 23

ags.org">↩<style type="text/css">↩block

File: txt2tags.py (hunk 3 above)
Comment (goetzschnell) [reply txt2tags.py]: Validator shows warning:
-----------------------------------------
Warning: Article lacks heading. Consider using h2-h6 elements to add identifying headings to all articles.
From line 37, column 1; to line 37, column 9
</header>↩<article>↩↩<div
------------------------------------------
from https://dev.w3.org/html5/html-author/#the-article-element:
-------------------------------------------
4.3.4.4 The article element
The article element represents an independent section of a document, page, or site. This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, or any other independent item of content.
--------------------------------------------
If the converted html page is a standalone document, it may contain an article but may not completely only an article.

File: txt2tags.py (hunk 4 above)
Comment (goetzschnell) [reply txt2tags.py]: The warning is:
---------------------------------------------------------
Warning: Consider using the h1 element as a top-level heading only (all h1 elements are treated as top-level headings by many screen readers and other tools).
From line 330, column 1; to line 330, column 4
<section>↩<h1>Specia
---------------------------------------------------------
in the w3 spec I did not found an indication for this warning, so I think it is ok to start with h1

File: txt2tags.py (hunk 1 above)
Comment (goetzschnell) [reply txt2tags.py]: eliminated

**Comment by goetzschnell** [comment]
w3 validator shows 5 warnings with  h1 element on converted html file with this patched version.
changed file txt2tags.py uploaded

**Comment by goetzschnell** [comment]
The result of the test is:

[2020-02-12result-patch4-html.txt](https://github.com/txt2tags/txt2tags/files/4194938/2020-02-12result-patch4-html.txt)

**Comment by jendrikseipp** [comment]
Merged, thanks!
